Cargo.toml

```diff
@@ -133,6 +133,7 @@ inventory = "0.1"
 maxminddb = "0.13.0"
 strip-ansi-escapes = "0.1.0"
 colored = "1.9"
+notify = "4.0.14"
```

Comment:
@a-rodin you may know more about this but is there anything we should worry about using any of these notify crates?